### ISSUE TITLE
Add DiscordSuite for Discord Rich Presence integration

### ISSUE THREAD

**Issue by ds84182** [notcake/glua-minecraft-planning#17]
https://www.curseforge.com/minecraft/mc-mods/discordsuite

Discowd suite

featuwes

impwove gawme infowmation
dispways modpack infowmation
dispways cuwwent dimension
twacks ewapsed time pwayed

why?

whewn pwaying minecwaft usuawwy discowd might nowt even detect uwu pwaying iwt, with discowdsuite iwt enabwes uwu tuwu nowt onwy show thawt uwu'we pwaying minecwaft wewiabwy but awwow othew usews tuwu see mowe abouwt whawt uwu'we doing in the gawme.

**Comment by HotelCalifornia** [comment]
okay, so here me out

**Comment by HotelCalifornia** [comment]
d*mn I misspelled it

**Comment by HotelCalifornia** [comment]
anyway, hear me out. <details><summary>If you have ever seen a cat</summary>with balls you know that their not ugly hanging sacks of skin. They are the cutest little fucking puffballs that you've ever seen and you feel the urge to touch them. I can't really convey to you , glua-minecraft-planning, how soft and perfectly shaped these cat balls look. I was on a farm in Italy and they had cats wandering around the property and none of the guys were neutered so I went up to them and won their trust. I started by slowly petting behind the ears, you know, the stuff that like and can trust you for. After a while of that, I touched one of the soft cat testies. I know it's weird but it is really the cutest part of a cat. Anyways, I highly suggest touching cat balls.</details>
